samples/powerpoint/shapes/get-set-shapes.yaml

```diff
@@ -0,0 +1,227 @@
+order: 2
+id: powerpoint-shapes-get-set-shapes
+name: 'Get, set, load, and save shapes'
+description: Get and set one or more selected shapes. Load and save one or more shapes.
+host: POWERPOINT
+api_set:
+    PowerPointApi: '1.5'
+script:
+    content: |
+        $("#getSelectedShapes").click(() => tryCatch(getSelectedShapes));
+        $("#setSelectedShapes").click(() => tryCatch(setSelectedShapes));
+        $("#changeFill").click(() => tryCatch(changeFill));
+        $("#saveShapeSelection").click(() => tryCatch(saveShapeSelection));
+        $("#loadShapeSelection").click(() => tryCatch(loadShapeSelection));
+        $("#createShapes").click(() => tryCatch(createShapes));
+        $("#arrangeSelected").click(() => tryCatch(arrangeSelected));
+
+        async function getSelectedShapes() {
+          // Gets the shapes you selected on the slide and displays their IDs on the task pane.
+          await PowerPoint.run(async (context) => {
+            let finalTable = "";
+            const shapes = context.presentation.getSelectedShapes();
+            const shapeCount = shapes.getCount();
+            await context.sync();
+            finalTable += "<br>getSelectedShapes.getCount returned:<b>" + shapeCount.value + "</b><br>";
+            finalTable +=
+              "<br><table border=1 cellpadding=3 cellspacing=0><tr><td bgcolor=#3333EE><font color=white>Index</font></td><td bgcolor=#3333EE><font color=white>Id</font></td></tr>";
+            shapes.load("items");
+            await context.sync();
+            shapes.items.map((shape, index) => {
+              finalTable += "<tr><td>" + index + "</td><td>" + shape.id + "</td></tr>";
+            });
+            finalTable += "</table>";
+            $("#outputSpan").empty();
+            $("#outputSpan").append(finalTable);
+          });
+        }
+
+        async function setSelectedShapes() {
+          // Selects the first two shapes on the selected slide.
```

Comment:
This method selects the first two shapes on slide 1, not on the selected slide like this description says.

I ran this sample on a slide deck with two slides, and I happened to have the second slide selected when I started — so I added all the shapes to the second slide, and then this method didn't work because the method only selects the shapes if they're on slide 1. 

Issue 1: The method description doesn't match what the method does (but the html button text does match what the method does).  
Issue 2: Can the method get the shapes on the selected slide, instead of getting the shapes on slide 1?

Reply:
I changed to indicate this is for slide 1. I think the intent is to show how to get a specific slide through getItemAt(), so I'll leave it to get slide 1.